src/DiffEqOperators.jl

```diff
@@ -62,6 +63,7 @@ export AbstractDerivativeOperator, DerivativeOperator,
        CenteredDifference, UpwindDifference, nonlinear_diffusion, nonlinear_diffusion!
 export DirichletBC, Dirichlet0BC, NeumannBC, Neumann0BC, RobinBC, GeneralBC, MultiDimBC, PeriodicBC,
        MultiDimDirectionalBC, ComposedMultiDimBC
+export AxisymmetricSphereDomain
```

Comment:
can we make use of DomainSets.jl here? That's what I'm hoping to standardize around.

Reply:
Yeah this was only meant as a temporary hack until the DomainSets.jl interface is clearer - which seems like it would first require changes to [ModelingToolkit's `domain.jl`](https://github.com/SciML/ModelingToolkit.jl/blob/e1b8e40c37817643b34199431d56c4b70c7e77b4/src/domains.jl)? Since `VarDomainPairing` expects an `AbstractDomain`.

Reply:
Opened https://github.com/SciML/ModelingToolkit.jl/pull/976